### PR TITLE
[Task 3 | Sprint 4] POST /users/{user_id}/tasks with swagger documentation

### LINKED ISSUE #212: [Task 3] Fix the response codes for wrong input for POST /users/{user_id}/tasks
|Task 3| Fix the response codes for wrong input for POST /users/{user_id}/task and document the required input and expected output (Swagger in folder /swagger in the project – filename swagger-users.yaml)

### PR DESCRIPTION
**Pull request by Pabriones**
Fix the response codes for wrong input for POST /users/{user_id}/tasks and document the required input and expected output (Swagger in folder /swagger in the project – filename swagger-users.yaml)

### CODE REVIEW

**Review by RayWu222 — APPROVED** [review]

File: functions/HttpTriggerUserIdTask/__init__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import os
 import pyodbc
 import azure.functions as func
-from ..Utils.dbHandler import dbHandler
+from ..Utils.dbHandler import *
```
Comment (jlsnow301) [inline functions/HttpTriggerUserIdTask/__init__.py]: was this change still needed? also with the recent changes to this file, please merge upstream dev to make sure there are no conflicts

Comment (Pabriones) [reply functions/HttpTriggerUserIdTask/__init__.py]: not exactly, I was just trying to experiment to see if thats going to destroy the code , but its shouldnt be an issue.

**Review by sunny852206 — APPROVED** [review]
looks good to me!

**Review by jlsnow301 — APPROVED** [review]

File: functions/HttpTriggerUserIdTask/__init__.py (hunk 1 above)
Comment (jlsnow301) [reply functions/HttpTriggerUserIdTask/__init__.py]: Gotcha
